ga4gh/datamodel/__init__.py

```diff
@@ -508,6 +529,21 @@ def getParentContainer(self):
         """
         return self._parentContainer
 
+    def _scanDataFiles(self, dataDir, patterns):
```

Comment:
Seeing this method added in the diff suggests this pattern has been superseded by another one.  We should probably use a `populateFromFile` and/or `populateFromDirectory` method instead... thoughts, @jeromekelleher ?